File: client/src/app/shared/models/schedule.ts
import { ILoanApplication } from "./loan-application";
import { IScheduleRow } from "./schedule-row";

export interface ISchedule {
  title: string,
  totalToPay: number,
  totalInterestToPay: number,
  scheduleTable: IScheduleRow[],
  application: ILoanApplication
  
}

File: Core/Dtos/Schedule.cs
using Core.Entities;

namespace Core.Dtos
{
    public class Schedule
    {
        public string Title { get; set; }
        
        public float TotalToPay { get; set; }
        
        public float TotalInterestToPay { get; set; }
        
        public ScheduleRow[] ScheduleTable { get; set; }
        public LoanApplication Application { get; set; }
        
        
    }
}